infrastructure-playbooks/templates/logfile.j2

```diff
@@ -0,0 +1,14 @@
+Suggested cut and paste into "group_vars/osds.yml" for configuring with osd_scenario=lvm
+----------------------------------------------------------------------------------------
```

Comment:
It looks like we're missing the top level ``lvm_volumes`` here.

it should look like:

```
lvm_volumes:
  - data: foo
    journal: bar
 ...
```

Reply:
@andrewschoen It was decided with John Harrigan yesterday that `lvm_volumes:` wouldn't be a part of the output, since https://github.com/ceph/ceph-ansible/blob/master/group_vars/osds.yml.sample#L186 has an example already.